proxy-node-gateway/src/test/java/uk/gov/ida/notification/apprule/HubResponseAppRuleTests.java

```diff
@@ -133,18 +134,19 @@
     );
 
     private final Form postForm = new Form()
-            .param(SamlFormMessageType.SAML_RESPONSE, Base64.encodeAsString("I'm going to be a SAML blob"))
+            .param(SamlFormMessageType.SAML_RESPONSE, ValidationTestDataUtils.sample_hubSamlResponse)
             .param("RelayState", "relay-state");
 
     @Test
     public void hubResponseReturnsHtmlFormWithSamlBlob() throws Exception {
+        NewCookie sessionCookie = getSessionCookie(proxyNodeAppRule);
```

Comment:
Why pull this out to a variable?

Reply:
Reasonably sure I was debugging at one point and just wanted it on a clear line.